### PR TITLE
Leave out any possibility of any Tax bill

### PR DESCRIPTION
**Pull request by andrewgorash**
The original paragraph §IV.10 is in direct contradiction to paragraph §II.5 as any tax does not allow any mutualy agreeeable compensation by definition. From that could stem wide sort of constitution-interpretation problems allowing future governments to predate on citizens as they do now in any other country. Please, do NOT allow the Public Administration to levy taxes in any form what so ever. If you do, the whole concept and aim of Liberland is in vain as it will more sooner than later become ordinary predating state.

### CODE REVIEW

**Comment by Jean-LouisMesic** [comment]
Taxes are in all forms under the Liberland Constitution are absolutely voluntary per §II.5. §IV.10. is not excluded from this. All Tax Bills would constitutionally be voluntary contributions. Sales tax would be a voluntary addition to your bill. Land tax would be a voluntary contribution. Donations are voluntary contributions. Additionally, the public has the ability to veto on all tax bills.

Towards the bottom of Pull #13 @KacperZajc explains the voluntary tax well.

**Comment by liberlandcitizen** [comment]
@Jean-LouisMesic - that is NOT how I interpreted the words of this constitution or even the words of @KacperZajc who said:

"There is a mandatory referendum for any Bill that would introduce taxation. This was expressly approved by the President in my conversation with him. This safeguard amounts to voluntary taxation."

He went on to say:

"if the Assembly would like to levy other forms of taxes there is a mandatory referendum requirement - so it is also 'voluntary' in a sense that it can be blocked by the Citizens."

So tax bills, once passed by the assembly, approved by the voters in referendum (50%+1 to be approved) and ruled as constitutional by the Supreme Court would be law.  Everybody would be required to pay, unless somebody could please correct me.  

In summary, according to @KacperZajc, because there's a referendum and because a majority of citizens would have approved, it would be voluntary (though that doesn't meet my personal definition because almost 50% could say no, and yet would be forced to pay).

**Comment by mtelka** [comment]
I fully agree with @andrewgorash here.

**Comment by andrewgorash** [comment]
@Jean-LouisMesic: That is only one possible interpretation and even not very plausible one. Why would you need any "Tax bill" in order to collect _voluntary_  taxes? And even if the legislative power which will initially vote for constitution will have in mind your interpretation, it will not prevent later Assemblies or Justices of the Supreme Court to change mind. This particular provision is such a giant backdoor to the whole constitution, that it makes the rest nearly worthless. All it takes to create Leviathan is to approve 0,001 % compulsory tax as constitutional in the first place. After that it will be only matter of time to make it 1 %, 10 % etc. Just look to the US history. At first, federal tax was found unconstitutional. Just a few years later it was said, that in the case, that it is "small enough" it is constitutional. Hundred years later even 70% marginal federal tax rate was constitutional. The only way to prevent that is to outlaw any compulsory payments what so ever.

**Comment by andrewgorash** [comment]
As I have read it through #13 it is absolutely clear, that @KacperZajc does _not_ want to rely solely on voluntary taxes, making straw-man arguments about judicial system disappearing overnight. That is something, that obviously does not make any sense (unless people of Liberland wanted it that way because the judicial system would become overly corrupted), therefore the plan is, as I see it, to make _Liberland_ just a new ordinary country. The fact mentioned there, that _income_ taxes are prohibited is just a juggling with words as any tax  is actually an income tax, as M. Rothbard showed in his magnum opus Man, Economy, and State (https://mises.org/library/man-economy-and-state-power-and-market). There is no other way to pay tax than from you income.

**Comment by mtelka** [comment]
@andrewgorash: exactly!

On the other hand, there is so easy way how to avoid taxes at all.  For example, the government could provide sevaral paid services, like:

Citizenship: $1000 (one time payment)
Passport: $20/year
Residency: $1000/year
... and so on

**Comment by liberlandcitizen** [comment]
@mtelka Fees = taxes in my book.  Why are we so afraid of people being unwilling to donate voluntary taxes?  Do we believe people are greedy and selfish?  Do we believe that people don't care for their neighbor?  

If a young Libertarian government created a bill for water infrastructure, and put forth a proposal for paying for it...do you really think residents/citizens wouldn't contribute to such things?  Do you think they would rather truck in their water from Serbia and throw their human waste out their window?  

It's simple: voluntary taxes will come flying in for anything that is worthwhile.  We have the chance to create the most fiscally conservative and responsible government of all time.  But give the government the power to tax, and instead they will rob money from the pockets of the people and spend it recklessly, as all governments with this power tend to do.

**Comment by mtelka** [comment]
@liberlandcitizen: For the water infrastructure you do not need any law, any taxation.  Just free market is needed.  Water is product like every others are (like food, electricity, roads, watching a movie in cinema, PC, car, ... you name it).

Please change your mind:
tax == enforced fee for nothing (in general)
price == voluntary tax == voluntary fee for a service, or good, or whatever, you are not obliged/enforced to buy

BTW, the _voluntary tax_ is oxymoron.

**Comment by liberlandcitizen** [comment]
Of course it's an oxymoron.  We are better off calling it a donation.  At any rate, this is not the proper place to debate anarcho-capitalism vs. minarchism.  I'm not here for that at least.  Yes if no government existed the free market would likely take care of water infrastructure on its own.  I agree with you.

Let's please stay focused on what we are actually discussing which is the constitution as written by @KacperZajc and the President.  If Liberland is to be recognized, this constitution will be ratified.  Do you want it to include the power to tax or can you suggest a reasonable alternative that is acceptable to the people drafting the constitution?

**Comment by Jean-LouisMesic** [comment]
What about @joshmh 's canton idea? Could not the power to levy taxes be solely vested in non-geographic subgovernments, so the taxes would indeed be voluntary? Each canton would pay for its own portion of the police force, the courts, and general upkeep, and the amount would vary from canton to canton. There would also undoubtedly be cantons that relied solely on voluntary donations.

The problem here would be the non-levying cantons becoming leeches to federal funds, or even to other cantons.

**Comment by liberlandcitizen** [comment]
@Jean-LouisMesic -  maybe you missed my message above?  Or you disagree with it?  @KacperZajc has been charged by the Liberland team to draft a constitution.  He is being open and transparent about it to the point of putting it on github.  But we are here to comment and give feedback about THAT document....not to come up with other philosophies.  Right?  

Cantons?  That is a radically different form of government.  @KacperZajc has stated here very clearly that the constitution we have seen is VERY CLOSE to the constitution that Liberland will implement.  

So I beg of you.....and I'm probably too late to be commenting here (I have been involved with Liberland from the start though) PLEASE - let's stick to constructive feedback about the actual document!  Cantons - having no government - all of those are off topic and OFF THE TABLE at this point. 

Honestly there's a decent chance that nobody of authority is evening reading these comments anymore given how off topic we are!  And I am not dismissing your ideas in general.  Please, go find a terra nullius or a seastead and create cantons and I will move there!  For now we have Liberland!  

SO.....back on topic....

"§IV.10. A Taxation Bill proposing taxation no other than tax on actual land and/or sales tax shall be the only Bill capable of levying tax burdens in the Free Republic of Liberland" 

Can we offer REASONABLE and acceptable (to @KacperZajc and Vit) alternatives to this? I vote for making all taxes optional (donations).  Or something in the bill of rights that says you can not be prosecuted or otherwise punished for failure to pay taxes.  It's a simple addition that would allow the rest of this document to stay in place.  Thoughts?

**Comment by KacperZajc** [comment]
For now all I can say is that there is no settled opinion on the issue of taxation so I am really unable to answer your calls for changes but total ban on taxation is possible as well. Please do not demand any amendments now.

**Comment by liberlandcitizen** [comment]
@KacperZajc thank you for the update!  I am relieved to hear that getting rid of the taxes is at least under consideration.  I can speak on behalf of the LSA in telling you that pretty much all of us would like to see the taxes removed, for what it's worth.  

Anyway, thank you for putting so much time and thought into this constitution.

**Comment by Jean-LouisMesic** [comment]
@liberlandcitizen It's not necessarily off-topic, polycentric government has a lot of momentum on another thread. It's not radically different either, all it would take is the addition of an analog to the U.S. Constitution's tenth amendment establishing a second tier of government. But I digress. I will leave it to that thread for now. I was merely posing that it could be a potential solution for this specific problem.

**Comment by Jean-LouisMesic** [comment]
Additionally, +1 @liberlandcitizen for the idea of making failure to pay taxes an nonprosecutable offencse

**Comment by andrewgorash** [comment]
@liberlandcitizen: Thank you for the "stick to the point" post. If there is some urge (and I do not know why) to explicitly allow taxes in the Costitution, than it at least should be consistent with the rest of the document (therefore it is needed to settle the obviously mutually exclusive clauses II.5 and IV.10) and at the same time to strictly restrict the Administration in the amount (or proportion) of overall taxes and from making taxes selective in any form. 
Consider for example a bill which would tax only some specific parts of land or discriminate on the basis of the land size owned. However it is quite difficult to prevent this as there are nearly indefinite ways how to bypass it, if you want (for example you can tax all land by the same tax and at the same time incorporate "deductible" which would effectively make it "tax on the rich". And we all know how are these popular among masses. The same holds for proposed "sales tax". 
In my opinion, as I stated above, the only way, how to prevent the government from growing indefinitely is to forbid taxation at all and without exceptions, precisely in the way, how it does the paragraph II.5.

**Comment by KacperZajc** [comment]
OK guys Ive just talked to the President about taxes and here is what I can tell you:

For now we leave it as it is. So please do not pressure me into changing anything at this point.

In the future we consider:
1) abolishing all taxes altogether
2) abolishing sales tax (so land tax would be the only tax)
3) putting a cap on the amount that can be collected by the state in reference to a specified % of the GDP so that there is a constitutional upper limit of burdens that can be levied

We will get back to this discussion in the future.

**Comment by liberlandcitizen** [comment]
Well of course he's in favor of taxes now - he's meeting with the Bilderberg people :)  (just a joke)

**Comment by Jean-LouisMesic** [comment]
I'd be in favor of 2 and 3. Considering the low land area at our disposal, a land tax is reasonable to me.

Naturally, 1 is best, but we can't have everything, now can we. There's also always the possibility of doing this in the future.

**Comment by saintego** [comment]
Sorry for lame question, but how you'll count gdp without accountancy? OK you can guess it somehow...

**Comment by FellowTraveler** [comment]
Taxes for services (police, etc) should be City taxes, not Federal or Canton taxes.

You can have a city built on the Liberland 7 sq km and that city can have its own city taxes, and those can pay for the police in that city. There is no need for those taxes to be at the Federal level.

**Comment by liberlandcitizen** [comment]
Yikes @FellowTraveler now you have me worried about "lower" governments doing all of the things that this constitution forbids the "federal" government of Liberland from doing.  @KacperZajc does this constitution allow for this?

**Comment by Jean-LouisMesic** [comment]
@FellowTraveler brings up a good point, but polycentric law could also deal with this. It could be all part of land/property taxes in the city. Want to live in the city, pay for the public utilities and services. And nothing else.

We can also limit lower level governments using this document through a separate article, so worry not @liberlandcitizen .

**Comment by FellowTraveler** [comment]
By giving the city the power to tax land or sales, you can eliminate any need for the Federal government to have any taxes at all. Instead you can just apportion the revenue to run it from the cities and cantons. You can also create a "Senate" so the cities and cantons can have representation in their own federal deliberative body.

**Comment by terrorist96** [comment]
I like the idea of dual sovereignty. I would be in favor of giving the taxing power to the cities (or cantons - if that is accepted) as opposed to giving it directly to the feds. But then would each city/canton need its own constitution as well? Would cities/cantons have local legislative bodies? If so, it needs to be mentioned in the Constitution.
